doc/node.1

```diff
@@ -447,6 +447,15 @@ A glob pattern that excludes matching files from the coverage report
 .It Fl -test-coverage-include
 A glob pattern that only includes matching files in the coverage report
 .
+.It Fl -test-coverage-branches Ns = Ns Ar threshold
```

Comment:
Same comment in this file about the optional threshold on the existing experimental flag.